### PR TITLE
Set the JVM locale fix to en-US when running tests

### PR DESCRIPTION
**Pull request by wenleix**
This avoids different exception messages under different system
languages.

### CODE REVIEW

**Comment by wenleix** [comment]
This fixes https://github.com/prestodb/presto/issues/6655, as I verified the tests will succeed under Polish language setting.

However, when I set the Presto root pom.xml to use `60-SNAPSHOT` and run `mvn clean install`, I see the following error:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.0:compile (default-compile) on project presto-main: Compilation failure: Compilation failure:
[ERROR] /Users/wxie/Code/presto-all/presto/presto-main/src/main/java/com/facebook/presto/operator/scalar/ScalarFunctionImplementation.java:[62,17] reference to checkArgument is ambiguous
[ERROR] both method checkArgument(boolean,java.lang.String,java.lang.Object,java.lang.Object) in com.google.common.base.Preconditions and method checkArgument(boolean,java.lang.String,int,java.lang.Object) in com.google.common.base.Preconditions match
[ERROR] /Users/wxie/Code/presto-all/presto/presto-main/src/main/java/com/facebook/presto/operator/NestedLoopJoinPagesSupplier.java:[49,16] no suitable method found for transform(com.google.common.util.concurrent.SettableFuture<com.facebook.presto.operator.NestedLoopJoinPages>,com.google.common.util.concurrent.AsyncFunction<com.facebook.presto.operator.NestedLoopJoinPages,com.facebook.presto.operator.NestedLoopJoinPages>)
[ERROR] method com.google.common.util.concurrent.Futures.<I,O>transform(com.google.common.util.concurrent.ListenableFuture<I>,com.google.common.base.Function<? super I,? extends O>,java.util.concurrent.Executor) is not applicable
[ERROR] (cannot infer type-variable(s) I,O
[ERROR] (actual and formal argument lists differ in length))
[ERROR] method com.google.common.util.concurrent.Futures.<I,O>transform(com.google.common.util.concurrent.ListenableFuture<I>,com.google.common.base.Function<? super I,? extends O>) is not applicable
[ERROR] (cannot infer type-variable(s) I,O
[ERROR] (argument mismatch; com.google.common.util.concurrent.AsyncFunction<com.facebook.presto.operator.NestedLoopJoinPages,com.facebook.presto.operator.NestedLoopJoinPages> cannot be converted to com.google.common.base.Function<? super I,? extends O>))
[ERROR] /Users/wxie/Code/presto-all/presto/presto-main/src/main/java/com/facebook/presto/util/Threads.java:[62,27] cannot find symbol
[ERROR] symbol:   method get(com.google.common.util.concurrent.SettableFuture<java.lang.Boolean>,int,java.util.concurrent.TimeUnit,java.lang.Class<java.lang.Exception>)
[ERROR] location: class com.google.common.util.concurrent.Futures
...
```

Will check if it's an issue with the current airbase pom.xml .

**Comment by wenleix** [comment]
The guava is a separate issue and will be fixed by https://github.com/prestodb/presto/pull/6995

**Comment by findepi** [comment]
@wenleix any plans? Can we close https://github.com/prestodb/presto/issues/6655 ?

**Comment by wenleix** [comment]
@electrum : Any suggestions? Thanks!

**Comment by electrum** [comment]
Should we add properties for this? That would allow running tests multiple times for different locales. Otherwise, I’m fine to merge this. Please rebase and add CHANGES.

**Comment by findepi** [comment]
It think it should remain possible to run tests with different locales. For
example, we could want to run CLI or JDBC tests with different locales.

**Comment by ilfrin** [comment]
@wenleix is there anything blocking from merging this and releasing a new version?

**Comment by electrum** [comment]
Thanks for the reminder. I’ll merge and release when I arrive at the office.

**Comment by electrum** [comment]
Chatted with @wenleix and he's going to update it. I also wanted to know if it could use `<systemPropertyVariables>` rather than `-D` arguments.

**Comment by ilfrin** [comment]
@electrum this is great, lets get this in.
cc @wenleix 

thank you!

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+Airbase 85
+* Set the JVM locale fix to en-US when running tests
```
Comment (findepi) [inline CHANGES.md]: `Use en-US locale when running Surefire tests.`

Comment (electrum) [inline CHANGES.md]: Nit: add newline after version header

**Comment by findepi** [comment]
I am happy to see this approved. @wenleix could you please rebase?

**Comment by wenleix** [comment]
@findepi , @electrum : Done rebase. thanks!

File: CHANGES.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ Airbase 89
 * Dependency updates:
   - validation-api 2.0.1 (from 1.1.0)
   - BVal 2.0.0 (from 1.1.1)
+* Set the JVM locale fix to en-US when running Surefire tests.
```
[the remark this answers is not in the document]
Comment (findepi) [reply CHANGES.md]: "pin" ?

[the remark this answers is not in the document]
Comment (wenleix) [reply CHANGES.md]: "...fixed to en-US..." ?

[the remark this answers is not in the document]
Comment (martint) [reply CHANGES.md]: I think it should just say "Set the default JVM local for Surefire tests to en-US"

[the remark this answers is not in the document]
Comment (wenleix) [reply CHANGES.md]: Thanks @martint . Addressed.

**Comment by wenleix** [comment]
@electrum : Can you merge it ? Thanks!

**Pull request merged**